docs/advanced-features/instrumentation.md

```diff
@@ -0,0 +1,36 @@
+---
+description: Learn how to instrument your Next.js app.
+---
+
+> **Note**: This feature is experimental. To use it, you must explicitly opt in by defining `experimental.instrumentationHook = true;` in your `next.config.js`.
+
+# `instrumentation.ts`
+
+If you export a function named `register` from this file, we will call that function whenever a new server instance is bootstrapped.
+When your `register` function is deployed, it will be called continuously on each cold boot (but exactly once in each environment).
```

Comment:
this phrase is confusing

Reply:
Which exactly? line 9 or 10?

Reply:
Oh, you ment that `continuously` part. Removed that :+1: